dev-infra/commit-message/validate.ts

```diff
@@ -26,16 +26,29 @@ export interface ValidateCommitMessageResult {
 
 /** Regex matching a URL for an entire commit body line. */
 const COMMIT_BODY_URL_LINE_RE = /^https?:\/\/.*$/;
+
 /**
- * Regex matching a breaking change.
+ * Regular expression matching potential misuse of the `BREAKING CHANGE:` marker in a
+ * commit message. Commit messages containing one of the following snippets will fail:
  *
- * - Starts with BREAKING CHANGE
- * - Followed by a colon
- * - Followed by a single space or two consecutive new lines
+ *   - `BREAKING CHANGE <some-content>` | Here we assume the colon is missing by accident.
+ *   - `BREAKING-CHANGE: <some-content>` | The wrong keyword is used here.
+ *   - `BREAKING CHANGES: <some-content>` | The wrong keyword is used here.
+ *   - `BREAKING-CHANGES: <some-content>` | The wrong keyword is used here.
+ */
+const INCORRECT_BREAKING_CHANGE_BODY_RE =
+    /^(BREAKING CHANGE[^:]|BREAKING-CHANGE|BREAKING[ -]CHANGES)/m;
```

Comment:
[Nit] This could shortened as:

```suggestion
    /^(BREAKING CHANGE[^:]|BREAKING-CHANGES?)/m;
```

Reply:
Yeah, good point. I've played with simplifying this before but felt like being explicit is better here. Let me know if you feel strongly about changing this. Happy to change.